api/v2010/message/create-test-sms-messages-example-1/create-test-sms-messages-example-1.5.x.php

```diff
@@ -12,11 +12,8 @@
 $twilio = new Client($sid, $token);
 
 $message = $twilio->messages
-                  ->create("+5571981265131",
-                           array(
-                               "body" => "All in the game, yo",
-                               "from" => "+5571981265145"
-                           )
+                  ->create("+5571981265131", // to
+                           array("body" => "All in the game, yo", "from" => "")
```

Comment:
@po5i what happened to the `"from"` parameter value?

Reply:
I get the same output using `api-definitions` both `master` and `040e392`.
Same with yoyodyne `master` 🤔
Are you able to repro this?

Reply:
Looks like we don't have the "From" in the snippet definition. Go ahead an manually add it back into this snippet and we'll merge this PR. I'll fix the snippet definition.

Reply:
done!